website/utils/cdnURL.ts

```diff
@@ -0,0 +1,2 @@
+const CDN = 'https://d2va9gm4j17fy9.cloudfront.net';
+export default CDN;
```

Comment:
I think this could be reduced to `export default 'https://d2va9gm4j17fy9.cloudfront.net'`

Reply:
I think now what might make sense is to have a routes.ts with HOME_PAGE and CDN both being named exports from the same file.